src/Microsoft.AspNetCore.DataProtection.Redis/project.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": "1.1.0-*",
+  "description": "Microsoft Azure Blob storrage support as key store.",
+  "packOptions": {
+    "repository": {
+      "type": "git",
+      "url": "git://github.com/aspnet/dataprotection"
+    },
+    "tags": [
+      "aspnetcore",
+      "dataprotection",
+      "azure"
+    ]
+  },
+  "dependencies": {
+    "Microsoft.AspNetCore.DataProtection": "1.1.0-*",
+    "StackExchange.Redis": "1.1.604-alpha"
```

Comment:
Oh BTW, we do not want to refer to pre-release versions of packages. So for now you can refer to an older version. Yeah this will force you to use `net451` only. Following PR is keeping track of it: https://github.com/aspnet/Caching/pull/147